.travis.yml

```diff
@@ -115,11 +115,11 @@ script:
   - >
     if [[ ${TEST_TARGET} == 'default' ]]; then
       export IRIS_REPO_DIR=${INSTALL_DIR};
-      python -m iris.tests.runner --default-tests --system-tests --print-failed-images;
```

Comment:
Retire this testing infrastructure - it's flaky and pre-dates the existing workflow using perceptual image hashing.

If there are graphical failures on `travis-ci` then the developer will rerun locally, which is an acceptable workflow now that we have the necessary `idiff` capability and repeatable `travis-ci` environments locally. So this `--print-failed-images` capability is really no longer needed.